lib/config/defaults.js

```diff
@@ -323,7 +323,7 @@ const applyCacheDefaults = (
 			F(cache, "cacheLocation", () =>
 				path.resolve(cache.cacheDirectory, cache.name)
 			);
-			D(cache, "hashAlgorithm", "md4");
+			D(cache, "hashAlgorithm", "xxhash64");
```

Comment:
`experiments.futureDefaults` ?

Reply:
also it looks like output.hashFunction is used instead of `hashAlgorithm`..

Reply:
Since this is unused, could you remove that option in main

Reply:
I am wrong this is `cache.hashAlgorithm`, it is not the same as `output.hashFunction`.. but option `cache.hashAlgorithm` is unused in code (only defaults and normalization)

Reply:
Yep, I also think it's unused. We better remove it from the schema, then allowing a non-functional option that confuses people.